client/templates/globals/connectioninfo.html

```diff
@@ -1,31 +1,31 @@
 <template name="connectionInfo">
     {{#unless connectionStatus.connected}}
-        <div class="connection-info">
 
-            <div class="smallMarginBottom">
-                <span class="connection-warning warningsign">
-                    <a href="#" id="btnConnectionLost">
-                        <span class="glyphicon glyphicon-warning-sign"
-                              style="color: yellow; font-size: 200%;"
-                              title="Connection Lost!
-                   Status: {{connectionStatus.status}}
-                   Retry Count: {{connectionStatus.retryCount}}
-                   Retry Time: {{connectionWaitTime}} sec.
-                   Reason: {{connectionStatus.reason}}
-                   Click to reconnect."></span></a>
-                </span>
-            </div>
-
-            <span id="btnWarningExpandCollapse" class="glyphicon connection-warning btn-warning-expand" aria-hidden="true">
-                <p class="btn-warning-expand-text">i</p>
+            <span id="btnWarningExpandCollapse" class="connection-info btn-warning-expand" aria-hidden="true">
+                {{#if currentSymbol }}
+                    <span class="glyphicon glyphicon-warning-sign"></span>
+                {{else}}
+                    <span class="glyphicon glyphicon-remove remove-symbol"></span>
+                {{/if}}
             </span>
-            <div id="warningMessage" class="connection-warning connection-warning-text" >
-                You have lost connection to the 4Minitz server.
-                    You will lose unsaved data if you close your browser.
-                    Click icon for reconnection attempt.
-                    If problem persist, contact server admin.
+
+            <div id="warningMessage" class="connection-info connection-warning-text" aria-hidden="true">
+                You have lost the connection to the 4Minitz server.
+                If you close your browser, unsaved data might be lost.
+                Please contact your sever admin if this problem persists.
+                <a href="#" id="btnReconnect" class="btn-reconnect">
+                    <span class="btn btn-secondary btn-raised"
+                          style="color:yellow; font-size:100%; font-weight:bold; background:darkred;"
+                          title="Connection Lost!
+        Status: {{connectionStatus.status}}
+        Retry Count: {{connectionStatus.retryCount}}
+        Retry Time: {{connectionWaitTime}} sec.
+        Reason: {{connectionStatus.reason}}
+        Click to reconnect.">Try Reconnect
```

Comment:
I would suggest, to leave it "for the time being"... and merge it.
We can improve a little bit afterwards...
I will merge.